MonumentAddons.cs

```diff
@@ -3631,6 +3607,249 @@ private interface IDeepCollection
             return false;
         }
 
+        private class MonumentHelper
+        {
+            public DungeonEntranceMapper _dungeonEntranceMapper;
```

Comment:
Should be named PascalCase since it's public. Arguably `DungeonEntranceMapper` doesn't need to exist as a distinct class, but I'll leave that up to you for now (we're not following OOP very strictly).

Reply:
Since its just 2 methods, `DungeonEntranceMapper` refactored into `MonumentHelper` class directly